src/wagtail_ai/panels.py

```diff
@@ -113,7 +113,8 @@ def attrs(self):
             )
             attrs["data-wai-chooser-panel-limit-value"] = self.panel.suggest_limit
             attrs["data-wai-chooser-panel-vector-index-value"] = self.panel.vector_index
-            attrs["data-wai-chooser-panel-chunk-size-value"] = self.panel.chunk_size
+            if self.panel.chunk_size:
+                attrs["data-wai-chooser-panel-chunk-size-value"] = self.panel.chunk_size
```

Comment:
Alternatively, default the value to `0`, which would work as the agent does a falsy check:

https://github.com/wagtail/wagtail-ai/blob/77efebf211cff4c0b5a62a7e64ce3da014101194/src/wagtail_ai/agents/suggested_content.py#L64-L65